### DISCUSSION TITLE
Support for Hardware Acceleration on IMX8 chipset

### DISCUSSION THREAD

**Discussion by oneline99** [blakeblackshear/frigate#14470] [Ask A Question]
I am using a small device running on IMX8 chip (A53) which stated that it supports H264 and H265 in its spec sheet. (https://www.nxp.com/docs/en/fact-sheet/IMX8MMINIFS.pdf ) . I would like to find out whether it is possible to enable hardware acceleration arg for Frigate using this device. Appreciate your guidiance

**Comment by NickM-27** [comment]
You'd need to get a build of ffmpeg built with support for that

**Reply by oneline99** [reply]
Thanks for the response. Do you have any guidance on how to get the build ? Do i need to recompile the kernel or just rebuild the ffmpeg codes to support IMX8 ?

**Reply by NickM-27** [reply]
I'm not familiar with imx8 but you just need a statically built ffmpeg binary with that support

**Reply by oneline99** [reply]
Thanks Nick. I have built a static ffmpeg binary but unable to get it activated. I will check out whether there are other resources on this.
